README.md

```diff
@@ -1,63 +1,178 @@
+## Prerequisites
 
-All examples use `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` as the user that deploys and runs all tests. This address is funded by default when creating a local set of chains. Make sure it is funded on testnets as well. Run `node/printBalances` (optionally specify an address if you wish to check an address other than the one above) to check the balances on all five supported testnets of any address.
+### One-time setup
 
-## Deployment
+1. You'll need to have node.js installed to run network dapps. To make sure you have it installed, run `node -v`. If no version is returned, see [Nodejs.org/downloads](https://nodejs.org/en/download/).
 
-Some examples need to first be deployed on the network you wish to run them on. To do this simply run `node scripts/deploy examples/${name} ${local|testnet}`. Replace `${var}` appropriately. Cannot be run for the `sent-token` and `deposit-address` examples as no deployment is needed.
+2. Clone the repo with `git clone https://github.com/axelarnetwork/axelar-local-gmp-examples.git`.
 
-## Testing
+### Whenever you start a new work session
 
-All examples can be tested with `node scripts/test/{example-name} ${local|testnet} ...params`. See below for what values params can take. All params are optional and have default values and can be ommited.
+1. cd to `axelar-local-gmp-examples`.
 
-### Sent token
+2. Build contracts and tests (The update and install takes a few minutes.):
+   ```
+   npm update && npm install
+   npm run build
+   ```
 
-Run `node scripts/test examples/send-token ${local|testnet} ${source-chain} ${destination-chain} ${amount}` to send aUSDC from the source to the destination. To run on testnet you need to fund `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` with aUSDC and replace `local` with `testnet`.
+3. To connect to the axelar network, run `npm install axelarnetwork/axelar-local-dev`.
 
-### Deposit Address
+4. To run a local node, cd to `axelar-local-gmp-examples` in a separate terminal and run `node scripts/createLocal`. You’ll need to have this node running to deploy the dapps.
 
-Run `node scripts/test examples/deposti-address ${local|testnet} ${source-chain} ${destination-chain} ${amount}` to send aUSDC from the source to the destination. To run on testnet you need to fund `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` with aUSDC and replace `local` with `testnet`.
+5. To make sure that the address we use for examples is funded on all five supported testnets, run `node scripts/checkBalances`. We use `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` to deploy and test all examples. Alternatively, you can use `--address [<address>]` to specify a different address.
+
+## Deploy and test each example
+
+1. Each example has several variables. Enter a valid value in the format of the example.
+
+| Variable| Valid Values| Default| Example|
+--- | --- | ---| ---|
+|network|local, testnet|local?|local|
```

Comment:
@ggutoski 
- What is the default for the network, local? 
- What is the syntax for accepting the default? (You can't just leave it out. You'll get an error.) 
- For everything else besides deposit-address and send-token, do they have a choice or should they enter local?
- If they have a choice, what are the criteria? (General, such as: Only those comfortable using testnet to use it?)

Reply:
Each example should be runnable on both local and testnet, so yes the user has a choice in all cases.  (Do I understand your question correctly?)